### DISCUSSION TITLE
How to change the filter text inside the head filters?

### DISCUSSION THREAD

**Discussion by BennyAlex** [KevinVandy/material-react-table#677] [Q&A]
How can I change the text in the filter text field?
<img width="105" alt="image" src="https://github.com/KevinVandy/material-react-table/assets/17402724/1a5adb8b-f76b-4c81-a288-ed275609073a">

Thanks!

**Comment by zarchinard** [comment]
```jsx
<MaterialReactTable>
    muiTableHeadCellFilterTextFieldProps={{ placeholder: "your text" }}
</MaterialReactTable>
```
